doc/dev-guide/lvmo-units.md

```diff
@@ -0,0 +1,23 @@
+## LVM Volume Groups
+
+- *lvmVG* reconcile units deploys and manages LVMVolumeGroup CRs
+- The LVMVG resource manager creates individual LVMVolumeGroup CRs for each
+  deviceClass in the LVMCluster CR. The vgmanager watches the LVMVolumeGroup
+  and creates the required volume groups on the individual nodes based on the
+  specified deviceSelector and nodeSelector.
+- The corresponding CRs forms the basis of `vgManager` unit to create volume
+  groups and create lvmd config file
+
+## Openshift SCCs
+
+- When the operator is deployed on an Openshift cluster all the required
+  SCCs are created by `openshiftSccs` reconcile unit
+- The `vg-manager`, `topolvm-node` and `lvmd` containers need elevated
```

Comment:
Where is the RBAC defined? The SCCs don't define all the privileges.

Reply:
- yes, RBAC is either part of CSV or `make deploy`
- However, if you are referring to any security rbac, we are yet to tighten it as mentioned in #29

Reply:
I was primarily just referring to the RBAC so the operator and other controllers can have access to the K8s resources.

Reply:
The RBACs are part of the bundle (OCP) and manifests (k8s) in the config directory.